presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -815,6 +846,10 @@ example uses the earliest snapshot ID: ``2423571386296047175``
       testBranch | BRANCH | 3374797416068698476 | NULL                    | NULL                  | NULL
       testTag    | TAG    | 4686954189838128572 | 10                      | NULL                  | NULL
 
+Presto C++ Support
+^^^^^^^^^^^^^^^^^^
+
+All above extra hidden metadata tables, except `$changelog`, are supported in Presto C++.
```

Comment:
```suggestion
All above metadata tables, except `$changelog`, are supported in Presto C++.
```

Reply:
Here existing `extra hidden metadata tables` is taken from the main heading. But if we think we should just rephrase it saying - `All above metadata tables, except `$changelog`, are supported in Presto C++.` that's fine with me, LMK.